packages/loot-core/src/client/upgrade-notifications.ts

```diff
@@ -33,10 +32,9 @@ export default function checkForUpgradeNotifications(
             button: Platform.env !== 'mobile' && {
               title: 'Find schedules',
               action: async () => {
-                window.__history &&
-                  window.__history.push('/schedule/discover', {
-                    locationPtr: window.__history.location,
-                  });
+                history.push('/schedule/discover', {
```

Comment:
Shouldn't this use the new `navigate`?

Reply:
Yes, but the value for `locationPtr` is not available using `navigate`. Hence the section in the top comment about changing how that works